pootle/apps/contact/forms.py

```diff
@@ -18,10 +18,10 @@
 
 class ContactForm(MathCaptchaForm, OriginalContactForm):
 
-    field_order = ['name', 'email', 'subject', 'body', 'captcha_answer',
+    field_order = ['name', 'email', 'summary', 'body', 'captcha_answer',
```

Comment:
`email_subject` might be a less confusing name for this as I don't know anyone who call the subject of an email a summary.